tests/Aspire.Hosting.Milvus.Tests/Aspire.Hosting.Milvus.Tests.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <NoWarn>$(NoWarn);CS8002</NoWarn> <!-- Milvus.Client packages are not signed -->
+    <!-- Cannot be moved to helix as this uses TestProject -->
```

Comment:
What does "this uses TestProject" mean?

Reply:
It's because of this test:
https://github.com/dotnet/aspire/blob/357a0b2e35b9188ba7c4437be3287b9b86388176/tests/Aspire.Hosting.Qdrant.Tests/AddQdrantTests.cs#L167-L204
and
https://github.com/dotnet/aspire/blob/357a0b2e35b9188ba7c4437be3287b9b86388176/tests/Aspire.Hosting.Milvus.Tests/AddMilvusTests.cs#L95-L128

Maybe we can move these tests to `Aspire.Hosting.Tests` which runs on the build machine, which would allow Qdrant+Milvus ones to run on helix?

Reply:
Also, if this kind of a test is useful then we should add it for other `Aspire.Hosting.*` also.

Reply:
Yeah, i think we should rewrite this a bit. I'll put up a separate PR.

Reply:
This ended up being simple. TestProgram isn't being used. I pushed a change to this PR for it.

Reply:
nice! thanks 👍